New crusader and priest litanies

Macleod962 opened the pull request:
Added new litanies for priest
Added crusader's litanies
Added crusade group ritual, that reveals crusader litanies for believers
Added more reusable cooldowns for rituals, they can be personal now

NanakoAC commented:
Instead of making us figure it out from the code, can you please list all of the new rituals, what they do, and how they work

Macleod962 commented:
Four new priest's rituals just boost the stats (4 points) for personnel in view of the priest, even they haven't a cruciform (Special signal stimulate the brain). The effect lasts 2 minutes and have same cooldown.

New group ritual just opens crusader litanies. Depends on participants amount.

There are two crusader litanies:
Battle call. Increase robustness and toughness stats for user. Depends on believers in view. Lasts 10 minutes and have the same personal cooldown.
Flash. Stuns all unbelievers in a view and knocks the user. 2 minutes of personal cooldown.

NanakoAC commented on code/modules/core_implant/cruciform/rituals/crusader.dm:
```diff
@@ -0,0 +1,63 @@
+		if(brother.get_cruciform())
+			count++
+
+	user.stats.changeStat(STAT_TGH, count)
```
this is a pretty small buff, i'd say do count * 4

SpadesSlick replied:
That will snowball out of control. Remember that there are a lot of litanies that boost stats, so it's normal to have small numbers on each one.

SpadesSlick replied:
I think *2 should be more or less optimal.

NanakoAC commented on code/modules/core_implant/cruciform/rituals/priest.dm:
```diff
@@ -239,3 +239,97 @@
+
+/datum/ritual/cruciform/priest/short_boost/proc/take_boost(mob/living/carbon/human/participant, stat, amount)
+	participant.stats.changeStat(stat, -amount)
+	participant << SPAN_WARNING("You almost forgot something that you were got of [get_stats_to_text()]. The last piece of that knoweledge has disappeared.")
```
this english really needs revised

Macleod962 replied:
Should be better now

NanakoAC commented on code/modules/core_implant/cruciform/rituals/priest.dm:
```diff
@@ -239,3 +239,97 @@
+/datum/ritual/cruciform/priest/short_boost/mechanical
+	name = "Pounding Whisper"
+	phrase = "Vocavitque nomen eius Noe dicens iste consolabitur nos ab operibus et laboribus manuum nostrarum in terra cui maledixit Dominus"
+	stats_to_boost = list(STAT_MEC = 4)
```
4 is very little, especially for such a short duration
Try a value of 20 for all of these

SpadesSlick replied:
I think 20 will be too much. Let's stay on 10.

Macleod962 commented:
Updated
Added third crusader litany, that shows other cyberchristians with a special icon
Slightly fixed group litanies, they now change stats properly

NanakoAC commented on code/modules/core_implant/cruciform/rituals/crusader.dm:
```diff
@@ -0,0 +1,66 @@
+/datum/ritual/cruciform/crusader/brotherhood
+	name = "Eternal Brotherhood"
+	phrase = "Ita multi unum corpus sumus in Christo singuli autem alter alterius membra."
+	desc = "Reveals other cyberchristians to speaker."
```
I see two archetypes that would make this interesting:

1. Make it a sort of sweep scan. Only lasting for a couple of seconds, so you could use it to detect people in the same room

2. Make it burn most of your energy and last for a couple minutes, allowing a period of awareness
